stack-orchestration/configuration/di-account-interventions-integration-admin/ais-alarm-pipeline/parameters.json

```diff
@@ -0,0 +1,90 @@
+[
+  {
+    "ParameterKey": "AllowedAccounts",
+    "ParameterValue": "324281879537"
```

Comment:
I don't think production should be added here. It's staging to integration and production. But we do not want to promote to production yet.

There it's just in staging pipeline allow integration to read from the promotion bucket.

Reply:
And same will apply for other integration pipelines :)

Reply:
Awesome, thanks Dom will remove 🙌